content/hi/containerization.md

```diff
@@ -0,0 +1,17 @@
+---
+title: कंटेनरीकरण
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+कंटेनरीकरण एक एप्लिकेशन और उसकी निर्भरता को एक [कंटेनर इमेज](/container-image/) में बंडल करने की प्रक्रिया है। कंटेनर निर्माण प्रक्रिया के लिए [ओपन कंटेनर इनिशिएटिव](https://opencontainers.org) (OCI) मानक का पालन करना आवश्यक है। जब तक आउटपुट एक कंटेनर इमेज है जो इस मानक का पालन करती है, तब तक कोई फर्क नहीं पड़ता कि किस कंटेनरीकरण उपकरण का उपयोग किया जाता है।
```

Comment:
```suggestion
कंटेनरीकरण एक एप्लीकेशन और उसकी निर्भरता को एक [कंटेनर इमेज](/container-image/) में बांधने की प्रक्रिया है। कंटेनर निर्माण प्रक्रिया के लिए [ओपन कंटेनर इनिशिएटिव](https://opencontainers.org) (OCI) मानक का पालन करना आवश्यक है। जब तक आउटपुट एक कंटेनर इमेज है जो इस मानक का पालन करती है, तब तक कोई फर्क नहीं पड़ता कि किस कंटेनरीकरण उपकरण का उपयोग किया जाता है।
```

Reply:
@anubha-v-ardhan I guess `बंडल करने` is right here as it is a defined technical term. We can use `बांधने` in parenthesis.

Reply:
I think `बांधने` makes more sense here! Lets go with this @jayesh-srivastava